posthog/api/test/__snapshots__/test_insight_funnels.ambr

```diff
@@ -1,408 +0,0 @@
-# name: ClickhouseTestFunnelTypes.test_funnel_time_to_convert_auto_bins
```

Comment:
Why is this snapshot missing? The test still seems to be decorated with `@snapshot_clickhouse_queries`.

Reply:
Looks like it was some quirk of tests, the snapshot is not changed anymore